src/frontend/ProductCard/ProductCard.jsx

```diff
@@ -26,20 +33,40 @@ const ProductCard = () => {
                 </span>
               </span>
             </div>
-            <span className="price-card-product-price">{ratings}⭐️</span>
             <div className="price-card-body">{desc}</div>
             <div className="price-card-footer">
-              <button className="secondary-btn">Details</button>
-              <button className="primary-btn">
-                Buy
-                <span className="material-icons price-card-icon-cart">
-                  shopping_cart
-                </span>
-              </button>
+              <span className="price-card-product-price">{ratings}⭐️</span>
+              {cartState.cartItems.some((product) => product._id === _id) ? (
```

Comment:
We can extract this operation's result in some variable and then use it here as required. 
```javascript
const IsPresentInCart = cartState.cartItems.some((product) => product._id === _id
```